corehq/apps/domain/models.py

```diff
@@ -302,7 +302,9 @@ class Domain(QuickCachedDocumentMixin, BlobMixin, Document, SnapshotMixin):
 
     name = StringProperty()
     is_active = BooleanProperty()
-    date_created = DateTimeProperty()  # expected to be a naive datetime specified in UTC
+    # date_created is expected to be a naive datetime specified in UTC
+    # Defaulting to a lambda rather than utcnow directly to make freezegun function. Not ideal
+    date_created = DateTimeProperty(default=lambda: datetime.utcnow())
```

Comment:
Yeah, I mention this in the comment above it -- unfortunately, if it's set directly to `utcnow` rather than a lambda, freezegun never seems to get the opportunity to mock it out, and the tests become reliant on real datetimes. I don't love modifying production code just to make it testable, but I also didn't think the difference was huge here

Reply:
:+1:  sorry I missed that comment and made you reiterate it.